pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/AggregationFunctionFactory.java

```diff
@@ -42,79 +47,87 @@ private AggregationFunctionFactory() {
   public static AggregationFunction getAggregationFunction(AggregationInfo aggregationInfo,
       @Nullable BrokerRequest brokerRequest) {
     String functionName = aggregationInfo.getAggregationType();
+    String argumentsString = AggregationFunctionUtils.getColumn(aggregationInfo);
```

Comment:
is this temporary for Distinct or permanent?

Reply:
This is for passing multiple args as one concatenated string in this PR, to avoid backward incompatible change in request.thrift for AggregationInfo. Open to suggestions if there is a trivial fix here that can be done in this PR. Otherwise, will address it separately in a backward compatible way.